packages/proxy/src/proxy.ts

```diff
@@ -66,13 +68,20 @@ import {
   makeTempCredentials,
   verifyTempCredentials,
 } from "utils";
+import { differenceInSeconds, formatISO, parseISO } from "date-fns";
 import { openAIChatCompletionToChatEvent } from "./providers/openai";
 import { ChatCompletionCreateParamsBase } from "openai/resources/chat/completions";
 import { importPKCS8, SignJWT } from "jose";
 import { z } from "zod";
 
+type CachedMetadata = {
+  cached_at: string;
```

Comment:
i was thinking we could just store the Date object itself--I believe its (de)serialization to JSON should Just Work?